### DISCUSSION TITLE
Thinking Cap #1

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/211A-ASIAAM-191A#2] [Assignments]
#### Note: responses submitted after Tuesday 1pm (6/29) will not be eligible for awards.

> ## What do you think are problems with the way we use data and GIS?

Post your [thinking cap #1](https://github.com/albertkun/211A-ASIAAM-191A/blob/master/Week_01/Materials/2_thinking_cap_1.md) response here.

**Comment by luigimagdamit** [comment]
I believe one of the main issues with data is how readily we take in easily digestible visualizations without much consideration of the source of the data. Clear and concise sourcing of data should be more of a common practice, in my opinion. I saw a Twitter post one time sourcing all the journals and articles they used for a post and I thought it was cool. I also believe that GIS data can be intentionally misleading when taken out of context, one example of this being the pipeline that Albert discusses in the lecture.

**Reply by edmondywen** [reply]
I definitely agree with your insight about the importance of understanding where data is being sourced from. It's cool to hear that there are at least some twitter users who actually cite their information

**Reply by abi-200** [reply]
Lack of source consideration is definitely an issue I see a lot on non-peer reviewed sources of data (mainly social media) as well - statistics can exclude key details or larger concepts, and without credible citations, it is easy for the average data consumer to use data to further solidify their own narrative. Like you, I have also seen some Twitter/Instagram threads/posts that cite their sources in their post, and I wish this could become a more mainstream practice. I wonder if this will actually take off as a practice though, considering that the goal of many social media pages is not necessarily to be a credible source so much as it is to gain views.

**Reply by ramirezirvin** [reply]
I agree on this topic, sometimes it is important to know where the source is coming from, to prove the accuracy and validity of the information that is being posted on social media. There usually is a lack of sources, and sometimes could be innacurate.

**Reply by eandrusenki** [reply]
I absolutely agree with you, especially in the past year and a half with a lot of social issues being talked about on social media and stuff like that. I think a lot of people post visuals or use stats that may have zero credibility to them. Another thing, I think a lot of people don't take the time to understand the data they are using so they post/quote/say it but it's out of context.

**Comment by abi-200** [comment]
One major issue I notice with the way we (the general public) use data is the lack of consideration given to the reliability of its source. Anecdotally, among my own peers, we all check for credibility in academic settings, but for social media charts, graphs, and tweeted statistics, unless the post creator themself provides citations (which occurs sometimes but not often), the credibility of this data may often go unchecked. If we wish to use this data to strengthen our own arguments, consider other viewpoints, or create projects/enact change, this particular problem of how we use data may further compound.
Another problem with the way we use data is the lack of transparency among its use in some cases. Large companies are able to collect data on our personal information, as well as on our most visited sites, interested, and other consumer-driven data in order to use it for profit, and this can be done without the individual’s explicit and educated consent. While in some cases this collection of mass data can be beneficial to society, in many ways, it has the potential to be exploited without explicit and updated regulations, since without data use transparency, malicious uses may become more likely.
One more issue with the way we use data has to do with its presentations; that is, sometimes, the same data can be represented or expressed in differing ways such as to support different conclusions, based on author biases or desired outcome (ex. Income disparity in the United States as in the reading).
The way we use GIS can also face similar problems with regards to presentation bias, but the way we use GIS can also pose a problem if the GIS software is not automatically updated/live. Without live updates, data used to create maps can become outdated and display false or misleading information. In addition, on a temporal scale, if GIS is used to create maps that show change over time, there may be problems with only showing changes over a specific period/missing the bigger picture, which can once again be misleading.

**Reply by edmondywen** [reply]
Your analysis about transparency and corporations is super interesting. With the rise of big tech companies and their overarching reach on personal information, this issue is extremely pertinent. I believe this is especially true considering that machine learning and other forms of AI are more prevalently used in everyday applications, so lack of transparency could drastically hinder our ability to critically analyze the biases embedded within the AI models and their impact.

**Reply by luigimagdamit** [reply]
I think your point about data collection is very important. In our current day and age, user data is probably one of the most valuable things to buy, since it is currently the best way for companies to appeal to consumers, as well as for them to gain insight into their consumer bases. Your point about data representation is also very interesting. One of the things we are warned about in statistics is fitting data to models, when it is not appropriate. The goal is usually to find a model that appropriately attempts to explain the data itself rather than the other way around.

**Reply by hrfrahn** [reply]
You made a really good point about fact-checking on social media; I know that I've come across countless infographics/posts that make big claims, and I tend not to do any fact-checking because it's generally my friends that are re-posting the information and I'm inherently biased towards believing them. I'm not sure what the best way to address this is, but maybe encouraging people posting/reposting that kind of infographic to add sources would be a good start.

**Reply by ramirezirvin** [reply]
The lack of privacy for data collection is a serious topic in my opinion, because in some cases they are able to access personal information and sell it in areas where you might not want to be out in the open.

**Reply by eandrusenki** [reply]
I think it's really important that you brought up people using data that can sort of be twisted to express the presenter's views and opinions and even how GIS data can be used in different contexts depending on how the author wants it to be viewed. I think it's imperative that we check people's work, we check their sources, we use academic articles to see the full scope, we se what other sources the author is using and where they might be getting data from. And I think an issue which can never fully be fixed is that everyone places their own biases even when they try hard not to. I think we can't escape human biases.

**Comment by edmondywen** [comment]
While data itself is innocuous, the people behind data collection and presentation are responsible for the issues in the ways we use data. Data and by extension cartography are responsible for capturing reality and as such is subject to the same problems – who is represented, which “concepts and theories” are embedded into the research methodology, and which ideas are excluded (Crampton & Krygier 2010). For instance, if the people conducting a study have no intention to properly highlight the opinions of the actual people behind the data, then it is extremely difficult to properly and fairly magnify people’s interests. One common example of bias altering the messaging of data is when the scale on the axes of a chart is altered to minimize or emphasize the impact of a certain statistic. A hyperbolic example of problematic GIS is a map trying to downplay the threat of sea level rise by creating a map that chooses to only showcase information about landlocked regions hundreds of miles from the ocean. Ultimately, data reveals the biases held by the people responsible for designing and collecting the data, and until society is truly equitable, we need to hold all forms of data visualization and interpretation under critical scrutiny.

**Reply by luigimagdamit** [reply]
I think your point about the intentions of those conducting studies is important. When a researcher voids community opinion about an issue and instead chooses their own methodology independent from the community, they are making an attempt at a solution that is slightly helpful and unfulfilling at best and harmful at worst. It also removes community voice from the study, which removes any aspect of empowerment.

**Reply by abi-200** [reply]
I think you make a very interesting point about how cartography is reflective of people's own biases, and this can hold true even if the map is meant to be purely factual/scientific. Your point about the creator's intentions is also important because as we have seen, visual manipulation can allow maps to strengthen certain arguments and can thus be used in an unintended critique. This could also tie into the fact that nowadays, mapmaking and GIS is much more community-based than before and not just confined to academia, which can be good such that it fosters more community voices in data maps but also negative such that it can open up room to even more biases. Lastly, your point on how data visualization and interpretation must be held under scrutiny until society is equitable is also very interesting to me because it essentially states that we must always be critical of the data we consume (our responsibility as ethical consumers). Whether or not a truly equitable society is feasible or a utopia is sometimes debated, but regardless of this I do believe it is true that we should always consume our data with questions in mind/keep in mind that the data may not show the entire picture.

**Reply by hrfrahn** [reply]
Your point about how data can reveal the bias of those collecting it/designing the experiment is a great one (and I echo it in my post). Whenever I see a chart/data visualization that doesn't seem to add up, it's often useful to look at who produced it and what ulterior motives they might have had in doing so; this can help identify possible biases that might have influenced how the data is displayed.

**Reply by cynthinee** [reply]
I think this is the real strength of community-based approaches to mapping. As an individual, we all hold onto our assumptions and hypotheses around a research question. Even in the best case scenario where we're really understanding of what these things are, we could still ultimately produce knowledge that is either not relatively truthful (and hopefully not flat out wrong or misleading). But when we're working with others, especially those of the community we're studying, we have a system of checks and balances that enable meaningful discussions and contestations to happen before the final map or article or whatever end product is produced and made widely available.

**Comment by hrfrahn** [comment]
I think that a lot of the time we treat "data" as something completely objective and immune to any form of bias, when it reality it's extremely easy (and common) for data to be biased (both by accident or on purpose). Data is superficially objective in the numerical sense; obviously, there's only one correct mean/median/mode for a given dataset, and these numbers can sometimes give us important insights into the world. However, often times, using data in this way ignores any possible bias in measurement or data collection that may have occurred. For example, SAT/ACT scores are commonly assumed to be a proxy for the intelligence of high schoolers, and many colleges (including UCLA when I applied, although I think they've changed this since then) base a large part of their admissions process on these test scores. This view, however, ignores the fact that there are MANY other factors besides "intelligence" that go into determining test scores; preparation, how good someone is at test-taking, etc. In fact, the strongest determinant of SAT score may be the [income of the test-taker's parents](https://www.cnbc.com/2019/10/03/rich-students-get-better-sat-scores-heres-why.html). This is just one example of how a seemingly "objective" measure doesn't really capture the measurement it sets out to.

**Reply by abi-200** [reply]
I think you bring up an important point on superficial objectivity of data, and this can be why it is so easy for people to manipulate certain statistics and "facts" to further their own bias and narratives - the numbers may not lie, but their framing can shift opinions. The proxy examples you bring up are very interesting because I wouldn't have considered such measures to be examples of bias, but I can see how they would be an example of a problem in how we use data now that I think more about it. This further highlights the importance of always having objectivity in the back of our minds when we read data and maps in order to be the most educated consumer we are able to be. This way, their is joint responsibility from both the map maker and the map reader as to how to interpret the data.

**Comment by ramirezirvin** [comment]
It can be noted that a way data is collected throughout our day-to-day life is through obviously our phone and the applications within the phone. Despite our privacy settings, there is always one way for the apps to track everything we do. For instance, I have my location services off for my AMC app, but that doesn't stop it from showing up on my notifications when I am near walking distance of one to suggest to watch a movie. There are even apps that don't even have the option to turn off your location services within the app, and still track where you are located, what other apps you consume. Data collection is important, but what if you prefer to opt-out and there is not even an option for that?

**Reply by perezsara59** [reply]
You make a really good point about data collection and wanting to opt out but not being able to and that's a huge issue. Sometimes its not just opting out of location tracking but also getting targeted ads, for instance googling lays chips then opening instagram and getting a lays ad. How much control do we as consumers really have over the data that is collected from us?

**Reply by emilysuh** [reply]
You bring up a great point about data collection! It is really troubling that our personal data is being monetized but we have so little control over how that gets used. There are lots of concerns around consent and privacy that are being actively evaded by the institutions that collect data.

**Comment by eandrusenki** [comment]
I think it's really nice that the public can provide data and GIS data for everyone to use, however, I think it can be a large issue because I don't know how everything uploaded for public use can be monitored. For example, a lot of data is posted to ArcGIS hub and while some may come from more reliable sources such as the National Parks Services, others were uploaded by a person with no credibility, and no sources from which their data came from. Making data and GIS sources so readily available and also having such a surplus of data, while helpful, I think has steered a lot of people a way from checking for credibility and opting towards ease. Using what ever data source, shapefile, graph, statistic shows up first and that we can use for our points of view.

**Reply by perezsara59** [reply]
I agree and want to add that it can also sometimes be difficult to find the origin of the data because of the surplus and possibilities of repeatability but its important to use reliable data in order for your own work to credible.

**Comment by perezsara59** [comment]
Intention is a big factor that can influence the way data and GIS are used/what they are used for. I think its important to gather data and using GIS to visualize that to share the information more clearly is great, but problems arise when a cartographer has their intentions set on a specific topic and want to see a certain outcome that they selectively choose data and/or leave out other important details just to get their point across. Using data/maps to lie or mislead people is a problem if/when we fail to consider reliability. Data and GIS also creates problems when you use them for surveillance. On one hand it can help find missing people or victims of human trafficking, on the other hand its a violation of privacy if done without consent and can leave someone susceptible to stalking.

**Reply by cynthinee** [reply]
I think you bring up a great point about confirmation bias and ethical research practices. This is where being self-aware and explicit about one's assumptions and positionality is super important so that we, as researchers, aren't falling into confirming our own hypotheses. I really appreciate the point about thinking about issues of privacy and consent in this kind of research and mapping practice.

**Comment by cynthinee** [comment]
Visualizations are meant to take data and provide a more digestible, easy-to-understand format for understanding data. I know for myself, I'm not well-versed in, say, reading and interpreting data in a simple Excel format. I supposed there is a difference in a mere data visualization and an ethical one. I think a responsible researcher or citizen must consider how to represent relative truth(s) in their presentations of data. There's also the issue of how data is captured in the first place. I believe understanding the conditions of knowledge production are crucial in understanding the data itself. The great thing about GIS is the ability to produce complex layers of data on top of each other and tell a more robust data story. As a librarian, I also think that data literacy curriculum is important towards both of these issues.

**Reply by emilysuh** [reply]
You bring up great questions about data literacy, ethics, and epistemology, and I'd be interested in hearing how your work as a librarian has shaped your views. I am also interested in pursuing library and archival science! I think libraries are so great but also often underutilized by the community, and I wonder how access and literacy could be improved to be more inclusive (for instance, most people don't know how to search a database, or that they can access free movies or music through libraries, etc.).

**Reply by ronnie-blanche** [reply]
Hi, Cynthia! I appreciate how you point out how understanding the knowledge production is crucial to comprehending data. A lot of the time there is vital information that is left out when we don't take a step back and take a look at how the information is gathered. Thus, data can easily be misunderstood or skewered if we don't take this step.

**Comment by emilysuh** [comment]
One issue I see with the use of data and GIS today is their role in surveillance. Police departments, for instance, use surveillance technology to ramp up racist criminalization with racial profiling. (See ACRE Campaign's work on the issue here: https://acrecampaigns.org/research_post/21st-century-policing/). ICE contracts private companies to provide databases of personal information, which is then used to target and deport immigrants (https://theintercept.com/2021/04/02/ice-database-surveillance-lexisnexis/). GIS is used by both police and ICE agents to disproportionately target specific communities located in low-income, racially segregated neighborhoods.

**Reply by ronnie-blanche** [reply]
Hey, Emily! Thank you for pointing out how a lot of our legal institutions use data and GIS to cause harm. Not a lot of conversation around this topic exists, and I think it's very crucial for more people to understand that larger systems, like the police, also take part in misconstruing data for unethical reasons. Given that there's definitely a large issue with how low-income and BIPOC communities are disproportionately targeted, how should we go about using data and GIS to empower and draw attention to these communities?

**Comment by ronnie-blanche** [comment]
An issue I've noticed with how we use data and GIS is in how easy it is to withhold vital information that provides deeper insight and context to the visualizations that we're seeing. My first encounter with this was when I a psychology teacher explained that a graph detailing how drug use was more rampant among African American and Latine communities in comparison to their white counterparts. Though the information was partially true, the author of the study left out how the information was taken from people that lived in city projects, which mainly consisted of low-income BIPOC occupants! This vital piece of information would totally account for why Black and Latine communities ranked higher than their counterparts, but would also discredit the study as it was not inclusive and did not reflect the entire truth. Because of how effortless it can be to misconstrue data like this, there should be more efforts to not only be completely honest and forthright in visuals but also more of an attempt to fully provide credible sources for these GIS and data.
